lib/ansible/plugins/become/dzdo.py

```diff
@@ -87,8 +87,8 @@ def build_become_command(self, cmd, shell):
 
         flags = self.get_option('become_flags') or ''
         if self.get_option('become_pass'):
-            self._prompt = '[dzdo via ansible, key=%s] password:' % self._id
-            flags = '%s -p "%s"' % (flags.replace('-n', ''), self._prompt)
+            self.prompt = '[dzdo via ansible, key=%s] password: ' % self._id
```

Comment:
Was the space needed or is this to just keep things similar to 2.7?

Reply:
space change was unnecessary

Reply:
that was just a diff i spotted and was speculating as the issue (was wrong, it was prompt)